modules/vscode/hm.nix

```diff
@@ -28,7 +28,9 @@ in {
       userSettings = {
         "workbench.colorTheme" = "Stylix";
         "terminal.integrated.fontFamily" = "'${monospace.name}'";
+        "terminal.integrated.fontSize" = builtins.floor (sizes.terminal * 4 / 3 + 0.5); # pt to px
         "editor.fontFamily" = "'${monospace.name}'";
+        "editor.fontSize" = builtins.floor (sizes.applications * 4 / 3 + 0.5); # pt to px
```

Comment:
Do these options set the font size (hopefully) everywhere in the appliction or only in those specific parts? The initial issue report mentioned more font size scenarios:

> ```nix
>     "customPresentationMode.overrides": {
>         "debug.console.fontSize": 26,
>         "editor.fontSize": 26,
>         "markdown.preview.fontSize": 26,
>         "terminal.integrated.fontSize": 26,
>     },
>     "debug.console.fontSize": 16,
>     "editor.fontSize": 16,
>     "markdown.preview.fontSize": 16,
>     "terminal.integrated.fontSize": 16,
> ```
>
> -- https://github.com/danth/stylix/issues/509#issue-2471240705

Maybe VSCode has a "global" font size option.

Reply:
Essentially, my question is whether the current Stylix module covers all relevant font cases inside VSCode.

Reply:
it should now cover all default settings. for some font sizes, they were set to 13 or 56 by default while the main "editor.fontSize" was set to 14, so i included those ratios over 14 in the font settings. "editor.fontSize" serves as the nearly-global font size option. I also found niche missed cases where font-family was different. all should be fixed now